codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/IdempotencyTokenMiddlewareGenerator.java

```diff
@@ -67,6 +67,17 @@ private void execute(
         });
     }
 
+    /**
+	 * Gets the sort order of the customization from -128 to 127, with lowest
+	 * executed first.
```

Comment:
One space back.

Reply:
Thanks missed that when updated comments,